README.md

```diff
@@ -135,19 +135,7 @@ If you still need help to get started, feel free to reach out on our [community
 
 ### ⚙️ To Develop Locally
 
-1. Install the development dependencies: `poetry install --with dev --with test`
-
-2. Install the pre-commit git hooks: `pre-commit install`
-
-3. Run `maturin develop` or `maturin develop --cargo-extra-args="--features=io-uring"` for using the experimental version of actix-web. This command will build the Robyn Rust package and install it in your virtual environment.
-
-4. Run `poetry run test_server`. This will run a server containing several examples of routes we use for testing purposes. You can see them at `integration_tests/base_routes.py`. You can modify or add some to your likings.
-
-You can then request the server you ran from an other terminal. Here is a `GET` request done using [curl](https://curl.se/) for example:
-
-```bash
-curl http://localhost:8080/sync/str
-```
+Build instructions have moved to [contribution guidelines](https://github.com/sparckles/Robyn/blob/main/CONTRIBUTING.md)
```

Comment:
Let's not do this. Please copy all the instructions here. 

The idea behind this is that - the readme should be as self-documenting as possible. I dislike READMEs that want me to open different links. I usually lose context in the process.